### ISSUE TITLE
[SUGGESTION] Making Hackintool UI more consistent on macOS Big Sur

### ISSUE THREAD

**Issue by GGorAA** [headkaze/Hackintool#104]
Hello @headkaze, I really love your app Hackintool. It really helped me in making my Hackintosh build. I installed Big Sur on separate partition, installed there Hackintool, and... UI doesn't look like it was for Big Sur. For example, let's take Safari settings and tab switcher. In safari, the selected item text and icon is blue color, when others are black. In Hackintool, text and icon color remains black after selecting. And there are more parts of Hackintool, that doesn't look like real native UI. Screenshots will be posted later, I am now on my main partition. When I will finish the work, I will boot back to Big Sur, and make screenshots.

**Comment by GGorAA** [comment]
Look at the top tab switcher here
![](https://user-images.githubusercontent.com/61655029/90104959-c7a63400-dd4d-11ea-9039-612683c0f372.png)
And then here
![](https://user-images.githubusercontent.com/61655029/90105017-e1477b80-dd4d-11ea-8893-6ae8b7c44b69.png)

At first, background is gray-ish in Hackintool, which looks good in pre Big Sur systems, but not on Big Sur. Background needs to be completely white. And, as I said, text and icon color in selected item need to be blue

**Comment by ultratiem** [comment]
Thanks for bringing this up.

HT still has a lot of unnecessary noise in the UI. And I have to update the icons for macOS 11. Also, moving past the segmented controller architecture to display information would be ideal, especially moving forward.

When I have time I'll try to come up with a more modern UI and leave it to @headkaze to implement if he wants (been really busy lately so haven't been able to do these little side projects). But I'm all for modernizing Hackintool before the M class kills Hackintoshes 😔

**Comment by GGorAA** [comment]
Thanks for responding :D

I also made a concept of a new icon(I know that it is shit, but, well, this is a concept):

![image](https://user-images.githubusercontent.com/61655029/99529124-00d2ed80-29a8-11eb-8b02-bd63b1eb511d.png)
 And the `.icns` file itself:

[Hackintool.icns.zip](https://github.com/headkaze/Hackintool/files/5559897/Hackintool.icns.zip)

**Comment by ultratiem** [comment]
There is one floating around on https://macosicons.com that is modelled after Keychain Access that will also work in the interim.

Personally I would like to veer away from a knife (quite a violent metaphor not to mention is we are treading on Swiss Army's IP).

**Comment by GGorAA** [comment]
Hackintool got an update recently. I think it's UI is a lil worse. There are a lot of tabs, and the whole window is just too big. And yes, the default size of that window also does not match with the one that contains every tab

**Comment by headkaze** [comment]
> 
> 
> Hackintool got an update recently. I think it's UI is a lil worse. There are a lot of tabs, and the whole window is just too big. And yes, the default size of that window also does not match with the one that contains every tab

All I did was change the window to remove the texture setting (so it will match Big Sur) and added a new icon. I haven't changed anything else including tabs or window size. So I'm not sure what you're talking about. Perhaps another screenshot would help?

**Comment by telepati** [comment]
Especially the title of the app moves to the icons in the dropdown menu. Please keep the title on top of the icons. If it isn't possible to do that my suggestion remove the title cause everyone knows the Hackintool and they can easily seeing the version number under the about menu. 

<img width="768" alt="Screen Shot 2020-11-18 at 23 42 01" src="https://user-images.githubusercontent.com/3812286/99586419-c5acda80-29f8-11eb-8ab0-ee359fb4f161.png">

**Comment by headkaze** [comment]
Yeah I don't like how it adds the text to the titlebar either. If you discover the setting to turn it off please let me know.

**Comment by headkaze** [comment]
> 
> 
> Thanks for bringing this up.
> 
> HT still has a lot of unnecessary noise in the UI. And I have to update the icons for macOS 11. Also, moving past the segmented controller architecture to display information would be ideal, especially moving forward.
> 
> When I have time I'll try to come up with a more modern UI and leave it to @headkaze to implement if he wants (been really busy lately so haven't been able to do these little side projects). But I'm all for modernizing Hackintool before the M class kills Hackintoshes 😔

I probably wouldn't bother doing any UI updates unless I can just roll them in as a pull request as I have very limited time for this project myself. Appreciate all the UI updates you've done in the past though!
